### PR TITLE
[PowerMock] LPS 151638

### PR DESCRIPTION
**Pull request by JanisZhang**

### CODE REVIEW

**Comment by JanisZhang** [comment]
Hi @tinatian @dantewang,

This pull is to remove PowerMock in modules/apps/dynamic-data-mapping-form-renderer.

https://issues.liferay.com/browse/LPS-151638

Thanks for checking.

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/test/java/com/liferay/dynamic/data/mapping/form/renderer/internal/servlet/DDMFormTemplateContextProcessorTest.java [hunk 1]
```diff
@@ -441,13 +444,18 @@ private void _setUpJSONFactoryUtil() {
...
-			LanguageUtil.isAvailableLocale(Matchers.any(Locale.class))
-		).thenReturn(
+		Mockito.doReturn(
 			Boolean.TRUE
+		).when(
+			language
+		).isAvailableLocale(
+			Matchers.any(Locale.class)
 		);
+		languageUtil.setLanguage(language);
```
Comment (dantewang) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/test/java/com/liferay/dynamic/data/mapping/form/renderer/internal/servlet/DDMFormTemplateContextProcessorTest.java:458]: @JanisZhang For format, usually we keep an empty line here (before line 458).

**Review by dantewang — APPROVED** [review]

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/test/java/com/liferay/dynamic/data/mapping/form/renderer/internal/servlet/DDMFormTemplateContextProcessorTest.java (hunk 1 above)
Comment (JanisZhang) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/test/java/com/liferay/dynamic/data/mapping/form/renderer/internal/servlet/DDMFormTemplateContextProcessorTest.java:458]: @dantewang Got it. I will force push, thanks!

**Comment by shuyangzhou** [comment]
See https://github.com/brianchandotcom/liferay-portal/pull/116291

**Comment by shuyangzhou** [comment]
ci:close

**Pull request closed without merging**